OData/test/E2ETest/WebStack.QA.Test.OData/DollarLevels/DollarLevelsTest.cs

```diff
@@ -194,5 +194,30 @@ public static void UpdateConfiguration(HttpConfiguration configuration)
             Assert.Contains(errorMessage,
                 result);
         }
+
+        [Theory]
```

Comment:
Add a case with 2 $levels in a query, like $expand=Manager($levels=max;$expand=DirectReports($levels=2))?